packages/core/src/server/helper.ts

```diff
@@ -47,6 +49,29 @@ const formatPrefix = (prefix: string | undefined) => {
   return `${hasLeadingSlash ? '' : '/'}${prefix}${hasTailSlash ? '' : '/'}`;
 };
 
+export const getRoutes = (options: { context: InternalContext }): Routes => {
+  return Object.entries(options.context.environments).reduce<Routes>(
+    (prev, [name, context]) => {
+      const distPrefix = relative(
+        options.context.distPath,
+        options.context.environments[name].distPath,
```

Comment:
Equal to environmentContext

```suggestion
        environmentContext.distPath,
```